FlagsmithClient/src/test/java/com/flagsmith/FeatureFlagCachingTests.kt

```diff
@@ -77,7 +77,7 @@ class FeatureFlagCachingTests {
         flagsmithWithCache = Flagsmith(
             environmentKey = "",
             baseUrl = "http://localhost:${mockServer.localPort}",
-            enableAnalytics = false,
+            enableAnalytics = true, // Mix up the analytics flag to test initialisation
```

Comment:
I think this has snuck in from [another PR](https://github.com/Flagsmith/flagsmith-kotlin-android-client/pull/40)?

Reply:
Can we merge the other PR first please so that we don't get mixed up @matthewelwell?

Reply:
I've merged it, but we squash and merge so you'll now need to rebase this PR.

Reply:
That's fine cheers